hathor/manager.py

```diff
@@ -87,6 +88,7 @@ def __init__(self,
                  consensus_algorithm: ConsensusAlgorithm,
                  peer_id: PeerId,
                  tx_storage: TransactionStorage,
+                 event_storage: EventStorage,
```

Comment:
Why would `HathorManager.__ini__(...)` receive an `EventStorage` object?

Reply:
Why don't you add methods to the EventManager to handle it? I mean, not sure we need to surface EventStorage to do this.

Reply:
Perfect. Let's do it in a separate PR so you can move forward with your other PRs.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/555